src/UeberMich.java

```diff
@@ -0,0 +1,15 @@
+public class UeberMich {
+    public static void main(String[] args) {
+
+        int age = 26;
+        String firstName = "Vanessa";
+        String lastName = "Tiziani";
+        char gender = 'W';
+        String birthday = "15.11.1993";
+        double averageGrades = 3.4;
+        boolean married = false;
+
+        System.out.println("Hello, my name is " + firstName + " " + lastName + ". I'm born on " + birthday + " and i'm " + gender + ". My Grades average is " + averageGrades + " and married= " + married + ".");
```

Comment:
Für die formattierte ausgabe schau dir bitte `printf` an. Damit kann man Platzhalten in Strings verwenden und muss nicht die Strings mit plus zeichen aneinanderketten